discovery-provider/src/tasks/index_nethermind.py

```diff
@@ -0,0 +1,1130 @@
+# pylint: disable=C0302
```

Comment:
I assume this file is mostly copy paste except for the few nethermind related lines (which I think we should call something besides nethermind still)